readme.md

```diff
@@ -124,6 +124,7 @@ Click the type names for complete docs.
 - [`Includes`](source/includes.d.ts) - Returns a boolean for whether the given array includes the given item.
 - [`Simplify`](source/simplify.d.ts) - Flatten the type output to improve type hints shown in editors.
 - [`Jsonify`](source/jsonify.d.ts) - Transform a type to one that is assignable to the `JsonValue` type.
+- [`Exact`](source/exact.d.ts) - Create a type from type A and B and convert all keys exclusive to type B as `never`.
```

Comment:
Thanks for the code review @sindresorhus 
I have updated the description to describe what it does instead of what it can do.

Reply:
You are now describing the actual logic of the code. The "what it does" needs to be more high level. Look at the other descriptions for inspiration.